docs/project_dads/about.md

```diff
@@ -15,7 +15,7 @@ The goal of this project is to create a flexible, web-based, search-driven user
 
 - **Source Code:** [https://github.com/oss-slu/dads](https://github.com/oss-slu/dads) [<img src="/img/git-alt.svg" alt="git" width="25" height="25" />](https://github.com/oss-slu/dads)
 - **Client** Dr Benjamin Hutz
-- **Current Tech Lead:** Abhilash Kotha [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/AbhilashKotha)  [<img src="/img/linkedin.svg" alt="linkedin" width="25" height="25" />](https://www.linkedin.com/in/abhilashkotha/)
```

Comment:
please move Abhilash to the Developers section, and add the labels "(alumni, prior tech lead)" for him. we want to continue recognizing all our contributors, even after they move on.